<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GifCreator</class>
 <widget class="QWidget" name="GifCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>350</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: transparent;
}

QLineEdit  {
	background-color: #3498db;
	color: #34495e;
	border: none;
}

QLineEdit:hover{
	background-color: #2980b9;
}

QSpinBox  {
	background-color: #3498db;
	color: #34495e;
}

QSpinBox:hover{
	background-color: #2980b9;
}
 
QSpinBox::up-button  {
  	background-color: #2c3e50;
	subcontrol-origin: border;
    subcontrol-position: top right; /* position at the top right corner */
 
    width: 16px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
}

 
QSpinBox::up-arrow  {
    image: url(:/Arrow/arrow_menu);
}
 
 
QSpinBox::down-button  {
    subcontrol-origin: border;
    subcontrol-position: bottom right; /* position at bottom right corner */
 
    width: 16px;
    background-color: #2c3e50;
    border-width: 1px;
    border-top-width: 0;
}

 
QSpinBox::down-button:pressed  {
    /*border-image: url(:/images/spindown_pressed.png) 1;*/
	background-color: #2980b9;
}
 
QSpinBox::down-arrow  {
    image: url(:/Arrow/arrow_menu_down);
}
 
QSpinBox::down-arrow:disabled,
QSpinBox::down-arrow:off  { /* off state when value in min */
   image: url(:/images/down_arrow_disabled.png);
}

</string>
  </property>
  <widget class="QWidget" name="widget_Top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  #34495e;</string>
   </property>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>0</y>
      <width>26</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>26</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>26</width>
      <height>18</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>26</width>
      <height>18</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
	image: url(:/Exit/exit_windows_os);
	background-color: none;
}
QPushButton:hover{
	image: url(:/Exit/exit_hover_windows_os);
}</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>18</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Bottom_Bar" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>313</y>
     <width>563</width>
     <height>32</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#Bottom_Bar{
background-color: #2c3e50;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="label_nbPictures">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #2980b9;</string>
      </property>
      <property name="text">
       <string>0 files</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Size">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Width">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_X">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string notr="true">X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Height">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Fps">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string notr="true">FPS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_Fps">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Record">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	image: url(:/GifCreator/record);
}

QPushButton:hover{
	image: url(:/GifCreator/record_hover);
}</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Record">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Pause">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	image: url(:/GifCreator/pause);
}

QPushButton:hover{
	image: url(:/GifCreator/pause_hover);
}</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Pause">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Stop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	image: url(:/GifCreator/stop);
}

QPushButton:hover{
	image: url(:/GifCreator/stop_hover);
}</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Stop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #3498db;
font: 8pt &quot;Open Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_Photo" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>21</y>
     <width>563</width>
     <height>292</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_Photo{
background-color: rgba(0, 0, 0, 25);
border: 2px solid rgba(0, 0, 0, 80);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_Working">
      <property name="font">
       <font>
        <family>Open Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ecf0f1;&quot;&gt;Working&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_borderLeft" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>353</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_borderBottom" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>345</y>
     <width>581</width>
     <height>5</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_borderRight" native="true">
   <property name="geometry">
    <rect>
     <x>567</x>
     <y>0</y>
     <width>4</width>
     <height>353</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e;</string>
   </property>
  </widget>
  <zorder>widget_Photo</zorder>
  <zorder>widget_Top</zorder>
  <zorder>Bottom_Bar</zorder>
  <zorder>widget_borderBottom</zorder>
  <zorder>widget_borderLeft</zorder>
  <zorder>widget_borderRight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
